PixiEditor/Views/Dialogs/HelloTherePopup.xaml.cs

```diff
@@ -22,7 +20,8 @@ public partial class HelloTherePopup : Window
         public static readonly DependencyProperty RecentlyOpenedEmptyProperty =
             DependencyProperty.Register(nameof(RecentlyOpenedEmpty), typeof(bool), typeof(HelloTherePopup));
 
-        public static string VersionText { get => $"v{Assembly.GetExecutingAssembly().GetName().Version.Major}.{Assembly.GetExecutingAssembly().GetName().Version.Minor}"; }
+        public static string VersionText => $"v{Assembly.GetExecutingAssembly().GetName().Version.Major}.{Assembly.GetExecutingAssembly().GetName().Version.Minor}" +
+                (Assembly.GetExecutingAssembly().GetName().Version.Build != 0 ? $".{Assembly.GetExecutingAssembly().GetName().Version.Build}" : "");
```

Comment:
You can assign `Assembly.GetExecutingAssembly().GetName()` to a variable, instead of calling the method 4 times.